test/BigInt/assign_by_value.js

```diff
@@ -0,0 +1,46 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+
+if (this.WScript && this.WScript.LoadScriptFile) { // Check for running in ch
+    this.WScript.LoadScriptFile("..\\UnitTestFramework\\UnitTestFramework.js");
+}
+
+var tests = [
+    {
+        name: "Assign BigInt literal",
+        body: function () {
+            var x = 123n;
+            var y = x;
+            assert.isTrue(x == 123n);
+            assert.isTrue(y == 123n);
+            x++;
+            assert.isTrue(x == 124n);
+            assert.isTrue(y == 123n);
+            y = x;

```

Comment:
I'm glad to see this test working again :)

Reply:
yeah, thanks to your suggestion 👍